test/ocTestStream.sh

```diff
@@ -1,55 +1,116 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
-# A recent version of ffmpeg is required for the loop of the provided videos
-# to repeat indefinitely.
-# Example: ./test/ocTestStream.sh ~/Downloads/*.mp4 rtmp://localhost/live/abc123
+# Requirements:
+#   ffmpeg (a recent version with loop video support)
```

Comment:
I didn't catch this before, but I thought one of the primary issues we were trying to resolve was people not having ffmpeg on their machine and not understanding how to install it. Can the script download ffmpeg (if not found) just like the other test scripts do? (Assuming it's run on Linux I guess).

Reply:
Our automated test scripts are run in a well-known environments (mostly Linux containers). If we want this script to be used by users who don’t know how to install a package on their own machine, we need to write a fool-proof solution which works everywhere. I don’t know a ready-made solution  for it. Also consider that this script runs on windows too (with cygwin).
What about pointing people to ffmpeg website with a note on how to find the right file instead?

Reply:
Isn't a fool-proof solution to help people work on Owncast what the original goal of https://github.com/owncast/owncast/issues/2502 was?

Reply:
In that case, I guess this PR is not addressing the issue! 😅 I’ll leave installing ffmpeg to people who are experienced in multi-platform scripting.